roles/build_dependencies/tasks/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: Install configured dependencies
   apt:
-    name: '{{ configured_dependencies }}'
+    name: '{{ common_dependencies + app_dependencies }}'
```

Comment:
whoa, you can just use `+`? that's pretty handy

Reply:
Yeah, that's what I found when I went looking for a solution! I think this is a lot simpler than whatever analogous thing I did before.